tests/scripts/pulp_rpm/test_rpm_sync_publish.sh

```diff
@@ -100,7 +100,7 @@ then
 fi
 
 expect_succ pulp rpm repository version list --repository "cli_test_rpm_repository"
-expect_succ pulp rpm repository version repair --repository "cli_test_rpm_repository" --version 1
+expect_succ pulp rpm repository version repair --version 1 --repository "cli_test_rpm_repository"
```

Comment:
Did you change __all__ of these?

Reply:
What do you mean? Just changed the order for 3 different test cases (also quite some time ago now)

Reply:
We want the old version to work nonetheless. So we should probably only change one instance.

> also quite some time ago now

This is an open PR, its all a projection into the future.